plugins/auth-auth0/src/utils/createAuthProperties.ts

```diff
@@ -31,13 +31,9 @@ export const DEFAULT_ROLE_LITERAL = builders.arrayExpression([
   builders.stringLiteral("user"),
 ]);
 
-export function generateRandomString(): string {
```

Comment:
Don't you need to remove the crypto import on line 10?

Reply:
done! Good catch :)